Unix/etc/omiserver.conf

```diff
@@ -23,7 +23,7 @@
 ##
 ## loglevel -- set the log level of the server
 ##
-loglevel = DEBUG
+loglevel = VERBOSE

```

Comment:
This should be either disabled altogether, or WARNING should be specified in the checked in version